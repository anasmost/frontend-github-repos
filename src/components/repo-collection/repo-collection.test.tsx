import { shallow } from "enzyme";
import RepoCollection from "./repo-collection";
import React from "react";

describe("<RepoCollection/> Test", () => {
  const repoCollectionWrapper = shallow(<RepoCollection />);
  it("initially renders a spinner with empty collection div", () => {
    expect(repoCollectionWrapper).toMatchSnapshot();
    expect(repoCollectionWrapper.prop("isLoading")).toBeTruthy();
  });

  // Elaborate additional tests for mounting stage
});
