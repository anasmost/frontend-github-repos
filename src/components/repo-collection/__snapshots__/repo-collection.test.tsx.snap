// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RepoCollection/> Test initially renders a spinner with empty collection div 1`] = `ShallowWrapper {}`;
